internal/bminventory/inventory.go

```diff
@@ -545,7 +545,6 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 			OpenshiftVersion:      *releaseImage.Version,
 			OcpReleaseImage:       *releaseImage.URL,
 			SSHPublicKey:          params.NewClusterParams.SSHPublicKey,
-			UpdatedAt:             strfmt.DateTime{},
```

Comment:
why is that removed?

Reply:
It is not needed since we set here default value.